main.cpp

```diff
@@ -791,6 +928,15 @@ void SynthApplication::modulateOneOscillatorWithAnother(OscillatorSection modula
                   << " has more than 1 voice, what may cause high CPU usage\n";
 }
 
+void SynthApplication::automateFilterQuality(FilterSection filterToAutomate)
+{
+    filterToAutomate.qualityFactor = 5.0f;
+    while (filter.qualityFactor < filterToAutomate.qualityFactor)
+        {
+            filter.qualityFactor += 0.5f;
+            std::cout << "Quality factor is " << filter.qualityFactor << std::endl;
+        }
```

Comment:
FIXME: indentation
curly braces should line up with the keywords on the line above

- [ ] revise